packages/amplify-cli/src/commands/gen2-migration/refactor/generators/template-generator.ts

```diff
@@ -33,7 +34,7 @@ import ora from 'ora';
 const CFN_RESOURCE_STACK_TYPE = 'AWS::CloudFormation::Stack';
 const GEN2_AMPLIFY_AUTH_LOGICAL_ID_PREFIX = 'amplifyAuth';
 
-const CATEGORIES: CATEGORY[] = ['auth', 'storage'];
+const CATEGORIES: CATEGORY[] = ['auth', 'storage', 'analytics'];
```

Comment:
Strings in this array are used to check against the prefix of category stacks in both Gen1 and Gen2. In Gen1, the analytics stack always starts with "analyticsamplify", but in Gen2, it can be any name. Codegen will need to handle the naming accordingly.